src/api/app/views/layouts/webui/webui.html.erb

```diff
@@ -39,6 +39,9 @@
     <%= csrf_meta_tag %>
   </head>
   <body>
+    <div class="overlay"></div>
+
+    <%= render 'peek/bar' %>
```

Comment:
Please remove this line as we don't have peek enabled in 2.8 (the reason for the test failures)